### ISSUE TITLE
v0.11.7 not compatible with v0.11.5 Mod on Linux

### ISSUE THREAD

**Issue by ArthurKChan** [tModLoader/tModLoader#903]
### Description
When I start the linux server, the mods are disabled.

### Log file
[server.log](https://github.com/tModLoader/tModLoader/files/4641242/server.log)

### Steps to reproduce
Environment: Nodecraft server running Linux. Testing with Magic Storage mod.

1. Ensure [enabled.json](https://github.com/tModLoader/tModLoader/files/4641267/enabled.zip) file has the mod enabled. located in `/.local/share/Terraria/ModLoader/Mods`
2. Start server

### Expected behavior 
Server starts with mod enabled.

### Actual behavior
get error:
```
An error occurred while loading MagicStorage
It has been detected that this mod was built for tModLoader v0.11.5
However, you are using tModLoader v0.11.7
The mod(s) have been automatically disabled.
```

### Reproduction frequency
everytime. 5 out of 5 times tried.

### Additional Information
v0.11.5 Mods + TModLoader v0.11.7 on a Windows server works fine.

**Comment by JavidPack** [comment]
Can you check the version of MagicStorage you have? I tested v0.4.3.5 and it worked on my linux server.

**Comment by ArthurKChan** [comment]
I am also using v0.4.3.5 of MagicStorage. Tried re-downloading the latest version and it still fails.

I am running into the same problem with all of the mods I have. I tried the same thing with just "AutoTrash" enabled and the same version incompatibility message comes up.
[server.log](https://github.com/tModLoader/tModLoader/files/4645668/server.1.log)

Since it worked for your linux server, it sounds like there is probably some setup issues with nodecraft's installation of the new tmod version or I screwed something up. I'll investigate further when I get the chance.
